examples/misc/lib/effective_dart/usage_good.dart

```diff
@@ -76,6 +76,33 @@ void miscDeclAnalyzedButNotTested() {
     // #enddocregion forEach-over-func
   };
 
+  {
+    var sequence = [];
+    // #docregion list-from-1
+    var copy1 = sequence.toList();
+    var copy2 = new List.from(sequence);
+    // #enddocregion list-from-1
+  }
+
+  {
+    var sequence = [];
+    // #docregion list-from-2
+    var sequence = [1, 2, 3]; // Creates a List<int>.
+    print(sequence.toList().runtimeType); // "List<int>".
+    print(new List.from(sequence)
+        .runtimeType); // "List" (List<dynamic>).
+    // #enddocregion list-from-2
```

Comment:
This should be tested.

Maybe change the `print()` to `assert()` statements?

We have tests using `print()` or `assert()` so it can be which ever you prefer.

Reply:
`print()` is easy for the reader to understand. `assert()` would be a lot harder because we can't do:

```dart
assert(sequence.toList().runtimeType == List<int>);
```

And I don't want to use an `is` there because then it confuses the issue with questions of how subtyping works with dynamic.

Is it really worth testing this?

Reply:
Up to you. Generally, whenever there is a `print()` in a code excerpt, we test it.

As I mentioned, I can push the code excerpt to the test file later.